Sources/ConsoleKit/Command/Commands.swift

```diff
@@ -6,14 +6,19 @@ public struct Commands {
     /// If set, this is the default top-level command that should run if no other commands are specified.
     public var defaultCommand: AnyCommand?
 
+    /// If `true`, an `autocomplete` subcommand will be added to any created `CommandGroup`.
+    public var enableAutocomplete: Bool
+
     /// Creates a new `ConfiguredCommands` struct. This is usually done by calling `resolve(for:)` on `Commands`.
     ///
     /// - parameters:
     ///     - commands: Top-level available commands, stored by unique name.
     ///     - defaultCommand: If set, this is the default top-level command that should run if no other commands are specified.
-    public init(commands: [String: AnyCommand] = [:], defaultCommand: AnyCommand? = nil) {
+    ///     - autocomplete: If `true`, an `autocomplete` subcommand will be added to any created `CommandGroup`.
```

Comment:
Mention that the `autocomplete` command will output a script that can be added to your shell configuration file to enable autocomplete for the command-line app.